os/dbuild.sh

```diff
@@ -168,7 +193,8 @@ function BUILD_TEST()
 {
 	# execute a shell script for build test
 	pushd ${OSDIR} > /dev/null
-	docker run --rm ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -w /root/tizenrt/os --privileged tizenrt/tizenrt:${DOCKER_VERSION} bash -c "./tools/build_test.sh"
+	CHECK_DOCKER_IMAGE ${DOCKER_VERSION}
```

Comment:
Thanks, if will docker version became same, I'll modify this.
and This could feel that late running dbuild.sh, because the menu does not appear immediately
 so I'll add to print "check docker image"

Reply:
Done